.buildkite/utils.sh

```diff
@@ -76,6 +76,32 @@ s/{{BUILDKITE_COMMIT}}/${BUILDKITE_COMMIT}/g;\
   kubectl rollout status deployment/conbench-deployment
 }
 
+upload_conbench_to_pypi() {
+  last_released_version=$(curl https://pypi.org/pypi/conbench/json | jq '.info.version' | tr -d '"')
+  version=$(cat version.py | tr -d '"' | tr -d '__version__ = ')
```

Comment:
Could this be used instead?

```
$ python -c "from conbench import __version__; print(__version__)"
1.37.0
```

We already have `__version__` defined here:

https://github.com/conbench/conbench/blob/8595f327f1d87ff8a0d58bbca56c704a7de8758b/conbench/__init__.py#L5

Reply:
Can't do this because using `from conbench import __version__` starts the whole app and I am getting errors:
https://buildkite.com/ursa-computing/conbench-deploy/builds/537#e068b114-b06a-45ce-a968-d606ec84c88c

I am going to try something else.

Reply:
Decided to use this: `version=$(grep version= setup.py | tr -d '"' | tr -d 'version=' | tr -d ',' | tr -d ' ')`